context/app/static/js/components/detailPage/style.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import { Alert } from 'js/shared-styles/alerts';
+import { entityHeaderHeight } from 'js/components/detailPage/entityHeader/EntityHeader';
+import Section, { baseOffset } from 'js/shared-styles/sections/Section';
+
+const DetailPageAlert = styled(Alert)`
+  margin-bottom: ${(props) => props.theme.spacing(2)}px;
+`;
+
+const DetailPageSection = styled(Section)`
+  scroll-margin-top: ${baseOffset + entityHeaderHeight}px;
+`;
+
+export { DetailPageAlert, DetailPageSection };
```

Comment:
I've put detail page specific versions of components from `shared-styles` in `js/components/detailPage/style` does this seem like an appropriate place?

Reply:
I think the placement is fine... I worry a little about future authors needing to remember that in this context they need to use particular version of the component, but I don't have a better idea off the top of my head.